### ISSUE TITLE
FB Summary page exact match on app name fails with PostgreSQL persistence layer

### ISSUE THREAD

**Issue by acspike** [orbeon/orbeon-forms#2081]
[From Twitter](https://twitter.com/orbeon/status/560968721548578816)

**Comment by acspike** [comment]
My configuration currently has two roles (OrbeonAdmin and OrbeonEducation) with a form-builder-permissions.xml like:

```
<roles>
    <role name="OrbeonAdmin"         app="*"             form="*"/>
    <role name="OrbeonEducation"     app="Education"     form="*"/>
</roles>
```

I created a form as in the Education app with the OrbeonAdmin role. The form was accessible to OrbeonEducation role if accessed directly but did not appear in the FB summary page or searches when using the PostgreSQL backend (MySQL worked). The reason was that the original xpath expression in the PostgreSQL persistence layer's exact match query was returning the element rather than the contained text (`<application-name>Education</application-name>` rather than `Education`).

If the testing process didn't catch this, it must not currently invoke the exact match branch. I don't know if there are other ways to test this, but anything that uses exact match should behave similarly.

Still learning how Orbeon fits together. Sorry to make this mistake and not know how to test it.

**Comment by acspike** [comment]
Fixed by #2082, can be closed.
